app/src/main/java/com/framgia/fsalon/screen/scheduler/detail/BookingDetailViewModel.java

```diff
@@ -641,12 +667,12 @@ private Uri getCaptureImageOutputUri() {
     /**
      * Define all side of customer 's photos
      */
-    @IntDef({SIDE_LEFT, SIDE_RIGHT, SIDE_BEHIND, SIDE_FRONT_UP})
+    @IntDef({SIDE_FIRST, SIDE_SECOND, SIDE_THIRD, SIDE_FOURTH})
     public @interface SideCapture {
-        int SIDE_LEFT = 0;
-        int SIDE_RIGHT = 1;
-        int SIDE_BEHIND = 2;
-        int SIDE_FRONT_UP = 3;
+        int SIDE_FIRST = 0;
+        int SIDE_SECOND = 1;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.